VesDeltaF: updated bibliography and other minor improvements

invemichele opened the pull request:
<!--
  Feel free to delete not relevant sections below
-->

##### Description

Minor updates to the VES_DELTA_F action:
- updated reference to the paper, that now has been published
- better explanation in doc about how to perform step one
- other minor improvements e.g. in the keywords, to be more consistent with the rest of plumed

Most probably this will be the final version.

##### Target release

<!-- please tell us where you would like your code to appear (e.g. v2.4): -->
I would like my code to appear in release v2.6

##### Type of contribution

<!--
  Please select the type of your contribution among these:
  (Change [ ] to [X] to tick an option)
-->
- [ ] changes to code or doc authored by PLUMED developers
- [x] changes to a module not authored by you
- [ ] new module contribution or edit of a module authored by you

##### Copyright

<!--
  In case you picked one of the first two choices
  MAKE SURE TO TICK ALSO THE FOLLOWING BOX
-->

- [x] I agree to transfer the copyright of the code I have written to the PLUMED developers or to the author of the code I am modifying.

<!--
  In case you picked the third choice (new module authored by you)
  MAKE SURE TO TICK ALSO THE FOLLOWING BOX
-->

- [ ] the module I added or modified contains a `COPYRIGHT` file with the correct license information. I also used the command `cd src && ./header.sh mymodulename` in order to make sure the headers of the module are correct.

##### Tests

<!--
  Make sure these boxes are checked. For Travis-CI tests, you can wait for them
  to be completed monitoring this page after your pull request has been submitted:
  http://travis-ci.org/plumed/plumed2/pull_requests
-->

- [x] I added a new regtest or modified an existing regtest to validate my changes.
- [ ] I verified that all regtests are passed successfully on Travis-CI.

<!--
  After your branch has been merged to the desired branch and then to plumed2/master, and after the
  plumed official manual has been updated, please check out the coverage scan at
  http://plumed.github.io/coverage-master
  In case your new features are not well covered, please try to add more complete regtests.
-->

valsson commented:
@invemichele there are a lot of changes to  src/lepton/..... which probably should not be there, right? 

Omar

invemichele commented:
yes, you are definitely right. I messed things up while syncing with master, and added back by mistake the old lepton asmjit folder. now I fixed it.
thanks Omar
